adagrams/game.py

```diff
@@ -1,11 +1,103 @@
+import copy
+import random
+    # create a list (ORIGINAL_POOL) with all the given letters
+    #LOOP:
+        # create deep copy of ORIGINAL_POOL (GAME_POOL)
+        # use random.choice(list) function to draw from pool (GAME_POOL) of letters
+        # make another list for player's hand (HAND)
+        # once letter is picked remove letter from pool list
+        # use while loop max the letters of player's hand to 10
```

Comment:
I appreciate that y'll started with pseudocode to work through the logic for `draw_letters()`!

Usually, comments left in production-ready code are to benefit future maintainers of your code (think docstrings with expected inputs and outputs or warnings for the next engineer). Remember to delete all your pseudocode and unnecessary comments before submission!